src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -242,16 +242,16 @@ public static double getFOVModifier(GameRenderer renderer, ActiveRenderInfo info
         //RenderingRegistry.registerBlockHandler(RenderBlockFluid.instance);
     }
 
-    public static void renderMainMenu(MainMenuScreen gui, MatrixStack mStack, FontRenderer font, int width, int height)
+    public static void renderMainMenu(MainMenuScreen gui, MatrixStack mStack, FontRenderer font, int width, int height, int alpha)
```

Comment:
Binary compatibility must be maintained here.

Reply:
Sorry I'm still a little new to contributing, what do you mean by this?